app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/validation.ts

```diff
@@ -2,21 +2,12 @@
 
 /** Regex pattern for valid AWS Cognito passwords. */
 export const PASSWORD_PATTERN =
-    '(?=.*[0-9])(?=.*[A-Z])(?=.*[a-z])(?=.*[ ^$*.\\[\\]\\{\\}\\(\\)?"!@#%&\\/,><\':;\\|_~`=+\\-])' +
-    '[0-9A-Za-z ^$*.\\[\\]\\{\\}\\(\\)?"!@#%&\\/,><\':;\\|_~`=+\\-]{6,256}'
+    '(?=.*[0-9])(?=.*[A-Z])(?=.*[a-z])(?=.*[ ^$*.\\[\\]\\{\\}\\(\\)?"!@#%&\\/,><\':;\\|_~`=+\\-]).{6,256}'
```

Comment:
Some explanation of this regex or a link to where it comes from might be nice.

Reply:
i was under the impression that we were using [cognito passwords](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-policies.html)

but the issue that this PR fixes, has non-basic-latin characters... so i might have to look into things further

Reply:
Indeed we are. I asked this question because I was confused. I had a gut feeling that this *shouldn't* be working, but it is. Cognito should be rejecting the password. But everything seems fine 🤷